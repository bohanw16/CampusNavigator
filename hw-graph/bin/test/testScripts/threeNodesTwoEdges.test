# threeNodesTwoEdges testing file
# Tests that a graph can be created,
# three nodes can be added to the graph,
# and two edges between them can be added

# Create a graph
CreateGraph g1

# Add three nodes
AddNode g1 n1
AddNode g1 n2
AddNode g1 n3

# Add two edges
AddEdge g1 n1 n2 e1
AddEdge g1 n2 n3 e2

# Print the nodes in the graph and the children of n1 and n2
ListNodes g1
ListChildren g1 n1
ListChildren g1 n2