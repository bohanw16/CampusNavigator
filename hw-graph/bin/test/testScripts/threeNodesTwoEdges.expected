# threeNodesTwoEdges testing file
# Tests that a graph can be created,
# three nodes can be added to the graph,
# and two edges between them can be added

# Create a graph
created graph g1

# Add three nodes
added node n1 to g1
added node n2 to g1
added node n3 to g1

# Add two edges
added edge e1 from n1 to n2 in g1
added edge e2 from n2 to n3 in g1

# Print the nodes in the graph and the children of n1 and n2
g1 contains: n1 n2 n3
the children of n1 in g1 are: n2(e1)
the children of n2 in g1 are: n3(e2)